commands/meme.js

```diff
@@ -0,0 +1,51 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "meme",
+        category: "Meme",
+        description: "Creates a meme, put an image URL for you background and then put the text you want along the bottom. Or put the image source afterwards. Who am I to judge?",
+        permissions: (msg) => msg.channel.type === 'dm' || msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
+        process: (msg) => {
+            let { suffix } = u.parse(msg, true);
+            //general globals from bot this was imported from
+            const args = suffix.trim().split(/\n/);
+            //Determine if a string is a url
+            function isURL(str) {
+                // Sloppy, but does the trick.
+                const url = /^<?(https?:\/\/\S*?)>?$/;
+                const match = url.exec(str);
+                return match ? match[1] : null;
+
+            }
+            let src;
+
+            if (msg.attachments.size > 0) {  // Message attachments as the default source
+              src = msg.attachments.first().url;
+            } else if (isURL(args[0])) { // Look for a URL at the beginning if one wasn't attached
+              src = isURL(shift(args));
```

Comment:
Gai issue in the earlier comments.
```suggestion
              src = isURL(args.shift());
```